src/v2/Components/ArtistSeriesRail/ArtistSeriesRail.tsx

```diff
@@ -63,7 +50,7 @@ const ArtistSeriesRail: React.FC<Props> = props => {
 }
 
 export const ArtistSeriesRailFragmentContainer = createFragmentContainer(
-  ArtistSeriesRail,
+  ArtistSeriesRail as React.FC<Props>,
```

Comment:
Is this needed?

Reply:
We are still not getting proper typings on relay components, so to have typings we do.

Reply:
Oh damn, I thought we fixed that :/